source/plugins/community/crypto/index.mjs

```diff
@@ -0,0 +1,73 @@
+//Setup
+export default async function({login, q, imports, data, account}, {enabled = false, extras = false} = {}) {
+  //Plugin execution
+  try {
+    //Check if plugin is enabled and requirements are met
+    if ((!q.crypto) || (!imports.metadata.plugins.crypto.enabled(enabled, {extras})))
+      return null
+
+        //Load inputs
+    let {id, days = 1, vs_currency = "usd", precision = 2 } = imports.metadata.plugins.crypto.inputs({data, account, q})
```

Comment:
You can remove the default values here, are they're already handled by the parser of `metadata.yml` so it avoids confusion

Reply:
ok